### PR TITLE
Update SupportBlock

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Update SupportBlock, add Intercom button and link for "phone onboarding"

https://user-images.githubusercontent.com/6661511/215120550-18900f1c-ca0d-42c8-b99c-0cb6ee2f3c71.mov

## Justify why they are needed
Make it possible to edit the content in Storyblok and add actions for each card.

Experimented a bit with the column layout we talked about with Zak @robinandeer but it ended up being a bit much code for simple cases like this. But let's se if Zak would prefer that 
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1458** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1458" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1458?utm_source=stack-comment).

File: apps/store/src/components/ContactCard/ContactCard.tsx [hunk 1]
```diff
@@ -1,53 +1,49 @@
...
 
-const Icon = styled.div(({ theme }) => ({
+const Icon = styled.div({
   display: 'grid',
   alignItems: 'center',
   justifyContent: 'center',
-  width: '76px',
-  height: '76px',
+  width: '80px',
+  height: '80px',
```
Comment (robinandeer) [inline apps/store/src/components/ContactCard/ContactCard.tsx:32]: Should this icon not scale with the content? Perhaps use rems

File: apps/store/src/components/ContactCard/ContactCard.tsx [hunk 2]
```diff
@@ -1,53 +1,49 @@
...
+  [mq.sm]: {
+    paddingBlock: theme.space.xxxl,
+  },
+})
 
-const Icon = styled.div(({ theme }) => ({
+const Icon = styled.div({
   display: 'grid',
   alignItems: 'center',
   justifyContent: 'center',
```
Comment (robinandeer) [inline apps/store/src/components/ContactCard/ContactCard.tsx:30]: nit: I think you can use "place-items" for this

File: apps/store/src/components/ContactSupport/IntercomChatButton.tsx [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
 import { useEffect } from 'react'
 import { IntercomProvider, useIntercom } from 'react-use-intercom'
-import { Button } from 'ui'
 
-const WithIntercom = () => {
+type Props = {
+  children: React.ReactNode
+}
+
+const IntercomButton = styled.div({
```
Comment (robinandeer) [inline apps/store/src/components/ContactSupport/IntercomChatButton.tsx:10]: Hmm... with "onclick" we should use button element or set the role attribute

**Review by robinandeer — APPROVED** [review]
Added via Giphy

Just left a few minor comments

File: apps/store/src/components/ContactSupport/IntercomChatButton.tsx (hunk 3 above)
Comment (gustaveen) [reply apps/store/src/components/ContactSupport/IntercomChatButton.tsx:10]: oops, didn't spot that one :D

File: apps/store/src/components/ContactCard/ContactCard.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/components/ContactCard/ContactCard.tsx:32]: Might be the case! Will update in next PR how the icons will work

**Pull request merged**